app/src/main/java/com/example/androidhomework/MainActivity.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.androidhomework
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.view.View
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        setupButton()
+    }
+
+    private fun setupButton() {
+        val play_button = findViewById<Button>(R.id.play_video)
```

Comment:
naming in code should be playButton. Underscore naming is only for xml
so correct would be `val playButton = findViewById<Button>(R.id.play_video)`